server/api/batch_inventory.py

```diff
@@ -39,7 +40,7 @@ class ElectionResults(TypedDict):
     ballot_count_by_batch: Dict[BatchKey, int]
     ballot_count_by_group: Dict[str, int]
     batch_to_counting_group: Dict[BatchKey, str]
-    # { batch_key: { choice_name: count } }
+    # { batch_key: { choice_id: count } }
     batch_tallies: Dict[BatchKey, Dict[str, int]]
```

Comment:
Note that, if we were to try to generate batch tallies using old batch inventory data persisted before this PR, we'd hit errors. But there isn't any way to do so for previously completed audits. So no need to run a migration.

Similar to an earlier PR, there is a question of whether this `batch_tallies` object should include the contest ID. But I ultimately didn't see a need for them when generating the actual batch-tallies / candidate-totals-by-batch files.